src/web/pages/reports/DeltaDetailsPage.tsx

```diff
@@ -0,0 +1,669 @@
+/* SPDX-FileCopyrightText: 2024 Greenbone AG
+ *
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+import {useState, useEffect, useMemo} from 'react';
+import {useDispatch, useSelector} from 'react-redux';
+import {useParams} from 'react-router';
+import logger from 'gmp/log';
+import Filter, {
+  ALL_FILTER,
+  RESET_FILTER,
+  RESULTS_FILTER_FILTER,
+} from 'gmp/models/filter';
+import Report from 'gmp/models/report';
+import Target from 'gmp/models/target';
+import {isActive} from 'gmp/models/task';
+import {first} from 'gmp/utils/array';
+import {isDefined, hasValue} from 'gmp/utils/identity';
+import Download from 'web/components/form/Download';
+import useDownload from 'web/components/form/useDownload';
+import Reload, {
+  NO_RELOAD,
+  USE_DEFAULT_RELOAD_INTERVAL_ACTIVE,
+} from 'web/components/loading/Reload';
+import DialogNotification from 'web/components/notification/DialogNotification';
+import useDialogNotification from 'web/components/notification/useDialogNotification';
+import useFilterSortBy from 'web/hooks/useFilterSortBy';
+import useGmp from 'web/hooks/useGmp';
+import useTranslation from 'web/hooks/useTranslation';
+import DeltaDetailsContent from 'web/pages/reports/DeltaDetailsContent';
+import ReportDetailsFilterDialog from 'web/pages/reports/DetailsFilterDialog';
+import DownloadReportDialog from 'web/pages/reports/DownloadReportDialog';
+import TargetComponent from 'web/pages/targets/Component';
+import {
+  loadAllEntities as loadFilters,
+  selector as filterSelector,
+} from 'web/store/entities/filters';
+import {loadDeltaReport} from 'web/store/entities/report/actions';
+import {deltaReportSelector} from 'web/store/entities/report/selectors';
+import {
+  loadAllEntities as loadReportConfigs,
+  selector as reportConfigsSelector,
+} from 'web/store/entities/reportconfigs';
+import {
+  loadAllEntities as loadReportFormats,
+  selector as reportFormatsSelector,
+} from 'web/store/entities/reportformats';
+import {pageFilter as setPageFilter} from 'web/store/pages/actions';
+import {
+  loadReportComposerDefaults,
+  saveReportComposerDefaults,
+} from 'web/store/usersettings/actions';
+import {loadUserSettingsDefaultFilter} from 'web/store/usersettings/defaultfilters/actions';
+import {getUserSettingsDefaultFilter} from 'web/store/usersettings/defaultfilters/selectors';
+import {loadUserSettingDefaults} from 'web/store/usersettings/defaults/actions';
+import {getUserSettingsDefaults} from 'web/store/usersettings/defaults/selectors';
+import {
+  getReportComposerDefaults,
+  getUsername,
+} from 'web/store/usersettings/selectors';
+import {generateFilename} from 'web/utils/Render';
+import {DESC} from 'web/utils/SortDirection';
+
+interface DeltaReportDetailsWrapperProps {
+  defaultFilter?: Filter;
+  reportFilter?: Filter;
+  entity?: Report;
+}
+
+interface DeltaReportDetailsProps {
+  deltaReportId: string;
+  entity: Report;
+  entityError?: unknown;
```

Comment:
I am currently defining errors as Error. Probably it can be a [Rejection instance](https://github.com/greenbone/gsa/blob/main/src/gmp/http/rejection.ts#L9) but the code handles that internally. Personally I would love if we could add some error type that is either `type ErrorType = Error | Rejection` or something like `interface ErrorType { message: string }`.
```suggestion
  entityError?: Error;
```